apps/client/components/Layout/TopNavigator/TopNavigator.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, Story } from "@storybook/react";
+import type { MouseEventHandler } from "react";
+import { Edit } from "~/assets/svgs/common";
+import { useNavigationSetter } from "~/recoil/atoms";
+import TopNavigator from ".";
+
+const Template: Story = ({
+  title,
+  right,
+  marginBottom,
+  hideBackButton,
+  backButtonCaution
+}: {
+  title?: JSX.Element;
+  right?: {
+    element: JSX.Element;
+    onClick: MouseEventHandler<HTMLButtonElement>;
+  };
+  marginBottom?: Common.CSSPixelValue;
+  hideBackButton?: true;
+  backButtonCaution?: boolean;
+}) => {
+  const setNavigation = useNavigationSetter();
+
+  setNavigation({
+    top: {
+      title,
+      right,
+      marginBottom,
+      hideBackButton,
+      backButtonCaution
+    }
+  });
+  return <TopNavigator />;
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  title: <div>This is title</div>,
+  marginBottom: 24,
+  hideBackButton: undefined,
+  backButtonCaution: true,
+  right: {
+    element: <Edit />,
+    onClick: () => console.log("clicked")
+  }
+};
+
+export default {
+  title: "TopNavigator",
+  component: TopNavigator
+} as Meta;
```

Comment:
```suggestion
export default {
  title: "TopNavigator",
  component: TopNavigator
} as ComponentMeta<typeof TopNavigator>;
```

6.5버전 에서는 Meta 타입말고 ComponentMeta를 사용하고 있는데 Meta를 사용하신 이유가 궁금합니다!

Meta는 스토리북 6.0버전 릴리즈 때 나왔고 ComponentMeta는 6.3버전 릴리즈 때 나온걸로 알고 있습니다!

Reply:
오 좋은 정보 감사합니다 ! 둘의 차이도 궁금하네요 일단 수정헀씁니다~~!